python/lsst/ts/mtdomecom/mtdome_com.py

```diff
@@ -780,6 +806,60 @@ async def has_non_status_command(self) -> bool:
         async with self._non_status_command_lock:
             return self._has_non_status_command
 
+    def is_moveAz_same_as_current(self, position: float, velocity: float) -> bool:
+        """Is the received moveAz command the same as the current moveAz
+        command or not.
+
+        Parameters
+        ----------
+        position : `float`
+            The target position to move to.
+        velocity : `float`
+            The velocity at which the target position is moving.
+
+        Returns
+        -------
+        bool
```

Comment:
bool -> `bool`

Reply:
No, return statements do not need to be quoted.